projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -58,7 +60,10 @@ import {combineLatest, Observable} from 'rxjs';
 import {map, pluck} from 'rxjs/operators';
 
 // TODO: remove in ivy compilation
-export const TIME_STREAM_FACTORY = TuiReplayControlValueChangesFactory;
+export const TIME_STREAM_FACTORY = (
+    control: NgControl | null,
+    valueTransformer: AbstractTuiControlValueTransformer<[TuiDay | null, TuiTime | null]>,
+) => TuiReplayControlValueChangesFactory(control, valueTransformer);
```

Comment:
This function name is way too long (and even starts with a capital letter), and so is the underlying `tuiReplayedValueChangesFrom`. We gotta rename them in 3.0 😄